src/Resources/views/CRUD/list_html.html.twig

```diff
@@ -7,9 +7,12 @@
         {%- if field_description.options.truncate is defined -%}
             {% set truncate = field_description.options.truncate %}
             {% set length = truncate.length|default(30) %}
-            {% set preserve = truncate.preserve|default(false) %}
+            {% if truncate.preserve is defined %}
+                {% deprecated 'The "truncate.preserve" option is deprecated since sonata-project/admin-bundle 3.x, to be removed in 4.0. Use "truncate.cut" instead.' %}
+            {% endif %}
+            {% set cut = truncate.cut is defined ? truncate.cut : (truncate.preserve is defined ? truncate.preserve != true : true) %}
```

Comment:
```suggestion
            {% set cut = truncate.cut is defined ? truncate.cut : not truncate.preserve|default(false) %}
```
I think this is more readable and it would also work if `truncate.preserve` is not defined.

Reply:
This was already defined in this way. The issue I see in that syntax is that `default` filter reads `false` as undefined; so regardless that syntax will work in this scenario, I'd prefer to be stricter about the definition of the option and its provided value.